File: KuantumJAVA/VeriPaketi.java
public class VeriPaketi extends KuantumNesnesi {

    public VeriPaketi(String ID, double stabilite, int tehlikeSeviyesi) {
        super(ID, stabilite, tehlikeSeviyesi);
    }

    @Override
    public void AnalizEt() {
        // Stabilite 5 düşecek
        setStabilite(getStabilite() - 5);
        System.out.println("Veri içeriği okundu.");
    }
}

    

File: KuantumJS2/VeriPaketi.js
const KuantumNesnesi = require("./KuantumNesnesi");

class VeriPaketi extends KuantumNesnesi {
    constructor(ID, stabilite, tehlikeSeviyesi) {
        super(ID, stabilite, tehlikeSeviyesi);
    }

    analizEt() {
        this.setStabilite(this.getStabilite() - 5);
        console.log("Veri içeriği okundu.");
    }
}

module.exports = VeriPaketi;
